less/about/preferences.less

```diff
@@ -455,6 +455,46 @@ table.sortableTable {
   overflow-y: scroll;
 }
 
+.modal .dialog.paymentHistory .sectionTitle {
+  text-align: left;
+}
+
+span.paymentHistoryButton.browserButton {
```

Comment:
this now looks awkwardly aligned with the select box but that can be saved for a follow-up issue.
![screen shot 2016-08-26 at 4 12 24 pm](https://cloud.githubusercontent.com/assets/549654/18023047/5ee1e924-6ba9-11e6-8e4b-7d5e21d3641a.png)